Makefile

```diff
@@ -2,8 +2,8 @@
 
 SOURCES := $(shell find src)
 
-target/fluree-json-ld.jar: pom.xml deps.edn src/deps.cljs $(SOURCES)
-	clojure -X:jar
+target/json-ld-0.1.0.jar: deps.edn src/deps.cljs $(SOURCES)
```

Comment:
This could probably come in another pr, but _while you're in here_, it'd be nice to make all of our Makefiles [self documenting](https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html). I've been doing that to each one as I've had to make other changes, but the only ones I've gotten to so far have been db, server, and nexus-server.